Sources/Droidy/Tools/SDK.swift

```diff
@@ -96,7 +96,7 @@ class SDK {
         print("sdkFolder.path: \(sdkFolder.path)")
         print("tools: \(cmdLineToolsBinFolder.appendingPathComponent("sdkmanager").path)")
         process.currentDirectoryPath = sdkFolder.path
-        process.arguments = ["bash", "-c", "yes | " + cmdLineToolsBinFolder.appendingPathComponent("sdkmanager").path + " --install \"platforms;android-\(droidy.project.compileSdkVersion)\" \"build-tools;\(buildToolsVersionToInstall)\" --sdk_root=" + sdkFolder.path]
```

Comment:
@Joannis are you sure that `build-tools` should be replaced with `platform-tools`?

Reply:
I didn't replace build-tools, I merely *added* `platform-tools`

Reply:
Those were not yet installed but requires for `adb`